1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -28,7 +28,8 @@ This topic provides all available configuration parameters. Except where explici
 | [master_discovery](#master_discovery)                                 | (Required) The Mesos master discovery method.         |
 | [platform](#platform)                                                 | The infrastructure platform. |
 | [public_agent_list](#public_agent_list)                               | A YAML nested list (`-`) of IPv4 addresses to your [public agent](/docs/1.10/overview/concepts/#public-agent-node) host names.  |
-| [rexray_config](#rexray_config)                                       | The [REX-Ray](https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/) configuration method for enabling external persistent volumes in Marathon. |
+| [rexray_config](#rexray_config)                                       | The [REX-Ray](https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/) configuration method for enabling external persistent volumes in Marathon. You cannot specify both `rexray_config` and `rexray_config_preset`.|
```

Comment:
Let's point to the docs for the REX-Ray version we've upgraded to:
https://rexray.readthedocs.io/en/v0.9.0/user-guide/config/